tuner/tuner/common.py

```diff
@@ -105,26 +110,63 @@ def is_comptible(mma_intrinsic: iree_gpu.MMAIntrinsic) -> bool:
     return list(filter(is_comptible, mma_intrinsics))
 
 
-class ReorderWorkgroupsStrategy(Enum):
-    NONE = 0
-    SWIZZLE = 1
-    TRANSPOSE = 2
-
-    def __str__(self) -> str:
-        return self.name.title()
-
-
 @dataclass
 class Configuration:
     subgroup_size: int
     workgroup_size: list[int]
-    intrinsic: iree_gpu.MMAAttr
-    tile_sizes: list[int]
-    subgroup_m_count: int
-    subgroup_n_count: int
+    lowering_config: iree_gpu.LoweringConfigAttr
     gpu_pipeline_options: iree_gpu.PipelineOptionsAttr
     waves_per_eu: int
 
+    def intrinsic(self) -> Optional[iree_gpu.MMAAttr]:
```

Comment:
If these helpers are useful enough, we could also add them to the iree python bindings. I think these helpers are also fine, but I'd rather see them as free functions instead of methods in `Configuration`. They don't need to know about `subgroup_size` and workgroup_size` after all -- the only input should the the lowering config attr.

Reply:
Good point, I will send another IREE PR to support this.